app/views/users/index.html.erb

```diff
@@ -1,2 +1,38 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+<%= render 'layouts/header' %>
+<div class="user_all_main_image">
+  <h1 class="user_all_title">Users</h1>
+</div>
+<div class="container pt50">
+    <div class="row">
+      <div class="col-lg-4">
+          <div class="user_details">
+              <div class="thumbnail">
+                  <%= image_tag @user.image.url, class: "user_image" %>
+              </div>
+              <p>Name:<%= @user.name  %></p>
+              <p>Email:<%= @user.email  %></p>
+              <p>lessons:4</p>
+              <p>Words_Numbers:125</p>
```

Comment:
@Masa08 Update `Words_Numbers` to `Words learned`. And `lessons` to `Lessons.`
Also add spaces after the colon `:`